erigon-lib/diagnostics/entities.go

```diff
@@ -155,6 +160,40 @@ type CPUInfo struct {
 	Mhz       float64 `json:"mhz"`
 }
 
+type BlockMetrics struct {
+	Header         *list.List
```

Comment:
choosing `Header` as a name is a bit too short in my opinion, please think about providing a bit more descriptive names